addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -3014,16 +3014,13 @@ StopMusic(client=0, bool:permanent=false)
 			{
 				StopSound(client, SNDCHAN_AUTO, currentBGM[client]);
 				StopSound(client, SNDCHAN_AUTO, currentBGM[client]);
-			}
 
-			if(MusicTimer[client]!=INVALID_HANDLE)
-			{
-				KillTimer(MusicTimer[client]);
-				MusicTimer[client]=INVALID_HANDLE;
-			}
+				if(MusicTimer[client])
```

Comment:
Please keep this outside of the `IsValidClient` check so that the timer is cleared regardless.